modules/java-modules/common/src/main/java/org/syslog_ng/logging/SyslogNgInternalLogger.java

```diff
@@ -58,15 +57,24 @@ public boolean requiresLayout() {
 
     @Override
     protected void append(LoggingEvent event) {
-        StringBuilder formatedMessage = new StringBuilder();
-        formatedMessage.append(event.getLocationInformation().getClassName());
-        formatedMessage.append(".");
-        formatedMessage.append(event.getLocationInformation().getMethodName());
-        formatedMessage.append(":");
-        formatedMessage.append(event.getLocationInformation().getLineNumber());
-        formatedMessage.append(" - ");
-        formatedMessage.append(event.getMessage().toString());
-        String message = formatedMessage.toString();
+
+        String message = null;
+        String internalMessageBody = event.getMessage().toString();
+
+        if (event.getLevel().isGreaterOrEqual(Level.INFO)) {
+            message = internalMessageBody;
+        }
+        else {
+            StringBuilder formatedMessage = new StringBuilder();
+            formatedMessage.append(event.getLocationInformation().getClassName());
```

Comment:
nitpicking: is not it cleaner to create a list for part of the formatted message and then loop over that list and append it to `formattedMessage`?